requirements.txt

```diff
@@ -12,13 +11,79 @@ urllib3==1.26.0
 pydantic==1.10.11
 httpx==0.25.0
 semver==3.0.2
-scipy==1.10.1
 memory-profiler==0.61.0
-cachetools==5.3.1
-pynvml==11.5.0 # For cuda utilization
-readerwriterlock==1.0.9
+pynvml==11.5.0
 kazoo==2.10.0
 pycurl==7.45.3
 certifi==2019.11.28
-transformers==4.41.2
-optimum==1.20.0
\ No newline at end of file
+transformers==4.31.0 # Consider 4.31.0 for _expand_mask
+
+# s2_inference:
```

Comment:
Where did all these new requirements come from?

Reply:
@farshidz  Got the new requirements from the languagebind repo https://github.com/PKU-YuanGroup/LanguageBind/blob/main/requirements.txt 

Had to upgrade/downgrade some existing ones to support languagebind. I would need your and @papa99do 's on these changes esp on the existing ones that were changed.

Reply:
done